digitalocean/spaces/resource_spaces_bucket.go

```diff
@@ -62,6 +62,7 @@ func ResourceDigitalOceanBucket() *schema.Resource {
 				Type:        schema.TypeList,
 				Optional:    true,
 				Description: "A container holding a list of elements describing allowed methods for a specific origin.",
+				Deprecated:  "Terraform will only perform drift detection if a configuration value is provided. Use the resource `digitalocean_spaces_bucket_cors_configuration` instead.",
```

Comment:
Adding deprecation flag to encourage users to use `digitalocean_spaces_bucket_cors_configuration`. AWS deprecated this to add more flexibility with bucket creations: 
https://github.com/hashicorp/terraform-provider-aws/issues/4418